### PR TITLE
sequence diagrams render as SVG in HTML

### PR DESCRIPTION
**Pull request by thomas-fossati**
Same as #418 but for sequence diagrams (as per popular demand)

### CODE REVIEW

**Review by henkbirkholz — CHANGES_REQUESTED** [review]
minor optics

**Review by nedmsmith — APPROVED** [review]

**Review by mcr — APPROVED** [review]

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -1719,28 +1731,31 @@ as discussed in {{epochfreshness}}, and the message is received during
...
         |   {I,time(EG_a)-time(VG_a)}     |               |
         |                |                |               |
         |                |           time(RG_v)           |
-        |<--Attestation Result------------|               |
+        |                |                |               |
+        |<--Attestation Result------------+               |
         |   {I,time(RX_v)-time(RG_v)}     |               |
         |                |                |               |
-     time(IR'_a)<-----I'-+--I'-------time(IR'_v)---->time(IR'_r)
+     time(IR'_a)<-----I'-o--I'-----> time(IR'_v) --> time(IR'_r)
```
[the remark this answers is not in the document]
Comment (mcr) [reply draft-ietf-rats-architecture.md]: I noticed the strange circle symbol in the SVG here, and I don't know what it means.

[the remark this answers is not in the document]
Comment (thomas-fossati) [reply draft-ietf-rats-architecture.md]: With the original '+' the horizontal line just passes through the vertical axis.  But the thing that (I think) needs to be shown here is the broadcast origin, which I pictorially made into a circle.  Like a lightbulb, or a tiny sun if you want.

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -1756,23 +1771,27 @@ solution that uses timestamps and requires roughly synchronized
...
+        ~                    ~                                 ~
+        |                    |                                 |
+  time(EG_a)                 |                                 |
+        |                    |                                 |
+        +----Evidence------->|                                 |
+        |   {time(EG_a)}  time(ER_r) ---Evidence{time(EG_a)}-->|
+        |                    |                                 |
+        |                    |                            time(RG_v)
+        |                    |                                 |
+        |                 time(RA_r)<--Attestation Result------+
```
[the remark this answers is not in the document]
Comment (thomas-fossati) [reply draft-ietf-rats-architecture.md]: Fixed in 246620c

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -1786,25 +1805,32 @@ are synchronized.
...
+     |                       |                             |
+  time(EG_a)                 |                             |
+     |                       |                             |
+     +----Evidence{Nonce}--->|                             |
+     |                       |                             |
+     |                    time(ER_r) ---Evidence{Nonce}--->|
+     |                       |                             |
+     |                       |                          time(RG_v)
+     |                       |                             |
+     |                    time(RA_r)<--Attestation Result--+
```
[the remark this answers is not in the document]
Comment (thomas-fossati) [reply draft-ietf-rats-architecture.md]: Fixed in 246620c

File: draft-ietf-rats-architecture.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (thomas-fossati) [reply draft-ietf-rats-architecture.md]: This one to was fixed in 246620c

[the remark this answers is not in the document]
Comment (mcr) [reply draft-ietf-rats-architecture.md]: I think that it's really the ~ that introduces visual disconnects that prevents understanding here.
I don't feel strongly though, so I've already marked approve. 
I await comments from @dthaler

File: draft-ietf-rats-architecture.md [hunk 4]
```diff
@@ -1719,28 +1731,31 @@ as discussed in {{epochfreshness}}, and the message is received during
...
+~~~~ aasvg
                   .-------------.
    .----------.   | Epoch ID    |   .----------.  .---------------.
    | Attester |   | Distributor |   | Verifier |  | Relying Party |
-   '----------'   '-------------'   '----------'  '---------------'
+   '----+-----'   '------+------'   '-----+----'  '-------+-------'
+        |                |                |               |
      time(VG_a)          |                |               |
         |                |                |               |
         ~                ~                ~               ~
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:1742]:
```suggestion
        ~                |                ~               ~
```

File: draft-ietf-rats-architecture.md [hunk 5]
```diff
@@ -1719,28 +1731,31 @@ as discussed in {{epochfreshness}}, and the message is received during
...
         |   {I,time(RX_v)-time(RG_v)}     |               |
         |                |                |               |
-     time(IR'_a)<-----I'-+--I'-------time(IR'_v)---->time(IR'_r)
+     time(IR'_a) <----I'-o--I'-----> time(IR'_v) --> time(IR'_r)
         |                |                |               |
-        |---[Attestation Result--------------------->time(RA_r)
+        +---[Attestation Result--------------------> time(RA_r)
         |   {I,time(RX_v)-time(RG_v)},I'] |               |
         |                |                |               |
         ~                ~                ~               ~
```
[the remark this answers is not in the document]
Comment (mcr) [reply draft-ietf-rats-architecture.md:1761]: Don't continue the Epoch line

File: draft-ietf-rats-architecture.md [hunk 6]
```diff
@@ -1719,28 +1731,31 @@ as discussed in {{epochfreshness}}, and the message is received during
...
         |   {I,time(EG_a)-time(VG_a)}     |               |
         |                |                |               |
         |                |           time(RG_v)           |
-        |<--Attestation Result------------|               |
+        |                |                |               |
+        |<--Attestation Result------------+               |
         |   {I,time(RX_v)-time(RG_v)}     |               |
         |                |                |               |
-     time(IR'_a)<-----I'-+--I'-------time(IR'_v)---->time(IR'_r)
+     time(IR'_a) <----I'-o--I'-----> time(IR'_v) --> time(IR'_r)
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:1756]: In the rendering that I'm seeing on my desktop, I see: 
  http://www.sandelman.ca/tmp/mcrcapture/2940.2022-06-13/capture1.png
note how I' to the left, but not I' to the right.  Bug in aasvg?

Comment (mcr) [inline draft-ietf-rats-architecture.md:1756]:
```suggestion
     time(IR'_a) <----I'-o--I' ----> time(IR'_v) --> time(IR'_r)
```

**Review by William-PanWei — APPROVED** [review]

**Pull request merged**